examples/coappearances.flatdata

```diff
@@ -0,0 +1,92 @@
+/**
+ * JSON Schema:
+ * {
+ *     "characters": {
+ *         "character_id": {             // two capital letters
+ *             "name": "String",
+ *             "nickname": "String",     // optional
+ *             "description": "String",  // optional
+ *             "relation": {             // optional
+ *                 "kind": "String",
+ *                 "to": "character_id or array of two character_ids"
+ *         }
+ *     },
+ *     "coappearances": [
+ *         {
+ *            "a": "character_id",
+ *            "b": "character_id",
+ *            "chapters": [
+ *                "#.##", ...             // # is a digit
+ *            ]
+ *         }
+ *     ]
+ * }
+ */
+namespace coappearances {
```

Comment:
The schema needs a few comments. Mostly, about the way things refer to each other. Otherwise presence of both "Relations" and "edges" doesn't make it straightforward to understand..

Reply:
Yes, make sense. I also added extended docs to the whole schema.